internal/providers/vmware/vmware_amd64.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/coreos/vcontext/report"
 	"github.com/vmware/vmw-guestinfo/rpcvmx"
 	"github.com/vmware/vmw-guestinfo/vmcheck"
-	"github.com/vmware/vmw-ovflib"
+	ovf "github.com/vmware/vmw-ovflib"
```

Comment:
Is this in the wrong commit?

Reply:
That's due to my vscode auto-formatting (Not sure why this change was even suggested). Fixed it.